platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -314,7 +342,12 @@ class HangingProtocolService {
 
       highestStudyMatchingScore = studyMatchDetails.score;
 
+      // TODO - use display sets instead of series, as that will use the
```

Comment:
Changing the matching here would allow interaction with the display sets, which means that proper regrouping rules could be defined, and then directly used by the hanging protocols.  That ultimately makes for much easier to define HPs because you don't have to specify both the regrouping rules as well as the hanging rules.  HOwever, compatibility with the existing DICOM HPs should be maintained, so need to think about how to do this properly.

Reply:
what do you mean by "interaction with display sets"?